### PR TITLE
feat: after downloading file, show it in folder instead of previewing it

### PR DESCRIPTION
**Pull request by dchest**
Files are no longer previewed after downloading, instead the destination directory is opened with system file manager with the downloaded file selected.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

#### Relevant issue tracker links  
...
#### Linked PRs in other repos

https://github.com/PeerioTechnologies/peerio-icebear/pull/88

#### Manual testing instructions  

1. Download some file.
2. Make sure it opens in system file manager after downloading with the file selected instead of showing its preview.

#### Additional notes
... 

----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner

- [ ] At least one approved code review exists.
- [ ] Approved tester review exists and was done after last code changes.
- [ ] CI build and tests passed.
- [ ] This feature/bugfix is considered fully done and can get merged.
- [ ] PR name follows conventional changelog format.
- [ ] I'll remember to squash commits.

### CODE REVIEW

File: src/helpers/file.js [hunk 1]
```diff
@@ -3,6 +3,16 @@ const fs = require('fs');
 const _ = require('lodash');
 const path = require('path');
 const sanitize = require('sanitize-filename');
+const util = require('util');
+
+function renameFile(oldPath, newPath) {
+    return new Promise((fulfill, reject) => {
+        fs.rename(oldPath, newPath, err => {
```
Comment (anri-asaturov) [inline src/helpers/file.js]: why not renameSync?

Comment (dchest) [reply src/helpers/file.js]: Don't wanna block, e.g. if spinning HDD has bad sectors and is trying to find a good one :)

Comment (dchest) [reply src/helpers/file.js]: Once `util.promisify` is in Electron, we can replace this with `renameFile = util.promisify(fs.rename)`.

Comment (anri-asaturov) [reply src/helpers/file.js]: How to detect an old man - he knows what 'bad sectors' are :-D
fs promisify is better to be done on a global level, so don't bother for now.

Comment (dchest) [reply src/helpers/file.js]: Haha. Well, I believe filled SSDs/flash drives still have delays when their controller is trying to find a place to write.

**Comment by dchest** [comment]
BTW, we should delete the .peeriodownload file if downloading files, right? Not sure how resume works currently or if it works at all.

**Comment by anri-asaturov** [comment]
Oh, crap, this will break resume. You have to put it in sdk.

**Comment by anri-asaturov** [comment]
Or maybe it won't break, let Eren test it first.

**Comment by dchest** [comment]
Yup, it will break. So no tests for now :)

**Comment by dchest** [comment]
I moved test steps to https://github.com/PeerioTechnologies/peerio-icebear/pull/89 and this PR is now only about opening folder for downloaded file instead of previewing it.

**Pull request merged**